source/draw/RasterBgr.ooc

```diff
@@ -94,8 +94,8 @@ RasterBgr: class extends RasterPacked {
 					if (c distance(o) > 0) {
 						maximum := o
 						minimum := o
-						for (otherY in Int maximum(0, y - this distanceRadius) .. Int minimum(y + 1 + this distanceRadius, this size y))
-							for (otherX in Int maximum(0, x - this distanceRadius) .. Int minimum(x + 1 + this distanceRadius, this size x))
+						for (otherY in 0 maximum(y - this distanceRadius) .. (y + 1 + this distanceRadius) minimum(this size y))
```

Comment:
Maybe there is no problem with readability, but `for (x in 0 maximum(...) .. y)` looks quite uncommon. Maybe it will take some time until I get used to it.